ros2_ws/src/otto_deliberation/include/otto_deliberation/tasks.h

```diff
@@ -86,4 +67,27 @@ class RobotTasks
     //! @param taskName task name
     //! @return task definition, emtpy task if not found
     Task ConstructTask(const RobotTaskKey& taskKey) const;
+
+    bool m_loop = false; //!< Whether to loop the tasks
+    RobotTaskList m_tasks; //!< Complete list of tasks
+    RobotTaskSet m_lifterTasks; //!< Tasks that need lifter in lifted position
+    RobotTaskSet m_dummyTasks; //!< Tasks that has robot in dummy mode
+    RobotTaskSet m_blindTasks; //!< Tasks that are blind (use blind path follower instead nav stack)
+    RobotTaskSet m_blindTasksReverse; //!< Tasks that are blind and need to be reversed
+    RobotTaskSet m_cargoLoadTasks; //!< Tasks that need to have cargo m_taskPaths to continue
+    RobotTaskSet m_cargoUnLoadTasks; //!< Tasks that need to have cargo un loaded to continue
+    RobotTaskSet m_acquireLock; //!< Tasks that needs acquire a lock
+    RobotTaskSet m_releaseLock; //!< Tasks that releases a lock
+    RobotTaskSet m_blindHighSpeed; //!< Tasks that are high speed blind
+    std::unordered_map<RobotTaskKey, double> m_postTaskDelays; //!< Tasks that need delay to start
+    std::unordered_map<RobotTaskKey, double> m_preTaskDelay; //!< Tasks that need delay to finish
+    std::unordered_map<RobotTaskKey, NavPathPtr> m_taskPaths; //!< Geometry of tasks
```

Comment:
RobotTaskSet and RobotTaskList were made because they are used as types in few other places. Type for paths and type for delays are used explicitly only here - for this reason I decided to not introduce aliases for them.